AdyenCard/Components/Card/CardViewControllerItemsProvider.swift

```diff
@@ -241,7 +276,23 @@ extension CardViewController {
             )
             return item
         }()
-        
+
+        internal func handleSelection(identifier: String?, brand: CardBrand? = nil) {
```

Comment:
Should you be able to "un-select" the brand? (Asking because of optional identifier)

Reply:
I have removed the identifier from this method.